doc/release-notes.md

```diff
@@ -0,0 +1,20 @@
+(note: this is a temporary file, to be added-to by anybody, and moved to
+release-notes at release time)
```

Comment:
Side-note: I'm proposing that we start including these in our release notes. Probably for now we can just have this file get prepended to the auto-generated release notes; we can edit the release notes generator to do so.